kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/ServiceControlCliSuite.scala

```diff
@@ -0,0 +1,327 @@
+    )
+    assert(getZkNamespace(new KyuubiCtlArguments(arg2)) == s"/${namespace}_USER/$user")
+  }
+
+  test("test list zk service nodes info") {
+    val uniqueNamespace = getUniqueNamespace()
+    conf
+      .unset(KyuubiConf.SERVER_KEYTAB)
+      .unset(KyuubiConf.SERVER_PRINCIPAL)
+      .set(HA_ZK_QUORUM, zkServer.getConnectString)
+      .set(HA_ZK_NAMESPACE, uniqueNamespace)
+      .set(KyuubiConf.FRONTEND_BIND_PORT, 0)
+
+    withZkClient(conf) { framework =>
+      createZkServiceNode(conf, framework, uniqueNamespace, "localhost:10000")
+      createZkServiceNode(conf, framework, uniqueNamespace, "localhost:10001")
+
+      val args = Array(
+        "list", "server",
+        "--zk-quorum", zkServer.getConnectString,
+        "--namespace", uniqueNamespace
+      )
+
+      // scalastyle:off
+      val expectedAns =
+        s"""
+          |+---------------------------------------------------------------------+----------+----------+--------------+
+          ||Service Node                                                         |HOST      |PORT      |VERSION       |
+          |+---------------------------------------------------------------------+----------+----------+--------------+
+          ||serviceUri=localhost:10000;version=$KYUUBI_VERSION;sequence=0000000000|localhost |10000     |$KYUUBI_VERSION|
+          ||serviceUri=localhost:10001;version=$KYUUBI_VERSION;sequence=0000000001|localhost |10001     |$KYUUBI_VERSION|
+          |+---------------------------------------------------------------------+----------+----------+--------------+
+          |2 row(s)""".stripMargin
+      // scalastyle:on
+
+      testPrematureExit(args, expectedAns)
+    }
+  }
+
+  test("test get zk service nodes info") {
+    val uniqueNamespace = getUniqueNamespace()
+    conf
+      .unset(KyuubiConf.SERVER_KEYTAB)
+      .unset(KyuubiConf.SERVER_PRINCIPAL)
+      .set(HA_ZK_QUORUM, zkServer.getConnectString)
+      .set(HA_ZK_NAMESPACE, uniqueNamespace)
+      .set(KyuubiConf.FRONTEND_BIND_PORT, 0)
+
+    withZkClient(conf) { framework =>
+      createZkServiceNode(conf, framework, uniqueNamespace, "localhost:10000")
+      createZkServiceNode(conf, framework, uniqueNamespace, "localhost:10001")
+
+      val args = Array(
+        "get", "server",
+        "--zk-quorum", zkServer.getConnectString,
+        "--namespace", uniqueNamespace,
+        "--host", "localhost",
+        "--port", "10000"
+      )
+
+      // scalastyle:off
+      val expectedAns =
+        s"""
+          |+---------------------------------------------------------------------+----------+----------+--------------+
+          ||Service Node                                                         |HOST      |PORT      |VERSION       |
+          |+---------------------------------------------------------------------+----------+----------+--------------+
+          ||serviceUri=localhost:10000;version=$KYUUBI_VERSION;sequence=0000000000|localhost |10000     |$KYUUBI_VERSION|
```

Comment:
is host/port/version redundant here? should it be `namespace | host | port | version`? and then we activate the pipeline with `delete` action?

Reply:
`Service Node` is not human-readable too

Reply:
done, now the format is.
```
        s"""
          |+----------------+----------+----------+--------------+
          ||   Namespace    |   HOST   |   PORT   |   VERSION    |
          |+----------------+----------+----------+--------------+
          ||/$uniqueNamespace|localhost |  10000   |1.2.0-SNAPSHOT|
          |+----------------+----------+----------+--------------+
          |1 row(s)""".stripMargin
```

Reply:
will  make a function to generate the expected ans,
otherwise, the ut might fail with any change of the fields.

Reply:
done